templates/addons/env/aurora-postgres.yml

```diff
@@ -15,7 +15,6 @@ Parameters:
     Default: main
     # Cannot have special characters
     # Naming constraints: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html#RDS_Limits.Constraints
-    "service": {{ service }}
```

Comment:
This shouldn't have been here, it's debug output left over from my previous pull request.